src/ui.text-alert/index.vue

```diff
@@ -8,15 +8,19 @@
       @binding {string} title
     -->
     <slot name="title" :title="title">
-      <div v-bind="titleAttrs" v-text="title" />
+      <div v-if="hasSlotContent($slots.title) || title" v-bind="titleAttrs" v-text="title" />
```

Comment:
fallback will be removed if `title` slot have data so this check seems to be redundant `hasSlotContent($slots.title)`